concent_api/middleman/middleman_server.py

```diff
@@ -72,19 +77,28 @@ def _run_forever(self) -> None:
 
     def _start_middleman(self) -> None:
         server_coroutine = asyncio.start_server(
-            self._handle_user_request,
+            self._handle_concent_connection,
             self._bind_address,
             self._internal_port,
             loop=self._loop
         )
         self._server_for_concent = self._loop.run_until_complete(server_coroutine)
+        server_coroutine = asyncio.start_server(
```

Comment:
Would it make sense to select different name than the same as above ?

Reply:
Done